spark/spark-tensorflow-connector/src/test/scala/org/tensorflow/spark/datasources/tfrecords/udf/DataFrameTfrConverterTest.scala

```diff
@@ -0,0 +1,133 @@
+package org.tensorflow.spark.datasources.tfrecords.udf
+
+import org.apache.spark.ml.linalg.SQLDataTypes.VectorType
+import org.apache.spark.ml.linalg.Vectors
+import org.apache.spark.sql.{Row, SparkSession}
+import org.apache.spark.sql.functions.{col, struct}
+import org.apache.spark.sql.types._
+import org.scalatest.{Matchers, WordSpec}
+import org.tensorflow.example.{Example, Feature, SequenceExample}
+import org.tensorflow.spark.datasources.tfrecords.TestingUtils._
+
+import scala.collection.JavaConverters._
+
+class DataFrameTfrConverterTest extends WordSpec with Matchers {
+
+  val sparkSession = SparkSession.builder()
+    .master("local[*]")
+    .appName("test")
+    .config("spark.driver.host", "localhost")
+    .config("spark.ui.enabled", value = false)
+    .getOrCreate()
+  val sparkContext = sparkSession.sparkContext
+

```

Comment:
Delete lines 16 to 22. The Spark session gets instantiated before you run the tests so you can either change line 53 and 103 to `spark.sparkContext.makeRDD(data)` or declare the `sparkContext` variable within the individual tests.

Reply:
Done